docs/tutorials/google_earth_engine/part_3/prepare.rst

```diff
@@ -96,7 +96,28 @@ Although using the :file:`gee/params.py` file to store our service account crede
 
     from . import params as gee_account
 
-4. **Delete** :file:`gee/params.py`.
```

Comment:
Seems like this file still needs to be deleted.

Reply:
This step was moved to step 5, it's still removed.